include/OpenColorIO/OpenColorIO.h

```diff
@@ -2976,27 +2977,17 @@ class OCIOEXPORT GpuShaderCreator
  *
  *    // Step 1: Create a GPU shader description
  *    //
- *    // The three potential scenarios are:
+ *    // The two potential scenarios are:
  *    //
  *    //   1. Instantiate the legacy shader description.  The color processor
  *    //      is baked down to contain at most one 3D LUT and no 1D LUTs.
  *    //
  *    //      This is the v1 behavior and will remain part of OCIO v2
  *    //      for backward compatibility.
  *    //
- *    OCIO::GpuShaderDescRcPtr shaderDesc
```

Comment:
Is this the intended change here? Currently this is removing the generic shader desc from docs and leaving legacy and custom only. Sould it be removing legacy and leaving gneric and custom instead?

And with this new implementation, the new legacy render path is just a generic shader desc with special optimization to replicate OCIO v1 behavior correct?

Reply:
The wrong description was preserved. The 'legacy' shader desc is now gone for v2.1.

> And with this new implementation, the new legacy render path is just a generic shader desc with special optimization to replicate OCIO v1 behavior correct?

Yes.

Reply:
Fixed